src/metatrain/cli/train.py

```diff
@@ -189,17 +191,30 @@ def train_model(
     train_datasets = []
     target_infos = TargetInfoDict()
     for train_options in options["training_set"]:
-        train_systems = read_systems(
-            filename=train_options["systems"]["read_from"],
-            fileformat=train_options["systems"]["file_format"],
-            dtype=dtype,
-        )
-        train_targets, target_info_dictionary = read_targets(
-            conf=train_options["targets"], dtype=dtype
-        )
-
-        target_infos.update(target_info_dictionary)
-        train_datasets.append(Dataset({"system": train_systems, **train_targets}))
+        if Path(train_options["systems"]["read_from"]).suffix == "":
```

Comment:
as a side note, I don't think we should do this based on file extensions. Instead, this feels like a good use case for #153 

One issue with using file extension is that some DFT software are creating files without extension (e.g. VASP), which we might want to be able to read in the future.

Reply:
Yes I talked to @frostedoyster and I am already on changing towards #153